pkg/server/message.go

```diff
@@ -309,15 +309,15 @@ func (t *TranslateServiceServer) fuzzyTranslate(
 			toBeTranslated.Messages = append(toBeTranslated.Messages, *msg)
 		}
 
-		// Translate messages -
-		// untranslated messages in toBeTranslated will be translated from original to target language.
+		// Translate translation -
```

Comment:
// Translate messages -
// untranslated messages in toBeTranslated will be translated from original to target language.